doc/contributing/writing-tests.md

```diff
@@ -139,6 +139,28 @@ request. Interesting things to notice:
   exit gracefully. Remember that for a test to succeed, it must exit with a
   status code of 0.
 
+## Running tests
```

Comment:
Not blocking this pull request but a suggestion for someone motivated to do it: This PR might contain information that we don't have elsewhere, but we do have "how to run tests" information in both BUILDING.md and doc/contributing/pull-requests.md. Not blocking on this pull request for this, but it would probably be good to consolidate how-to-run-tests info in one place and then put links to the One True Source Of Information in the docs that we remove it from. Part of me wants to suggest putting it all in `test/README.md`, but any of the places where it exists now would be fine, including the one being created in this PR.